Source/WebCore/rendering/style/BasicShapes.cpp

```diff
@@ -99,7 +99,7 @@ struct RoundedRectPathPolicy : public TinyLRUCachePolicy<FloatRoundedRect, Path>
     static Path createValueForKey(const FloatRoundedRect& rect)
     {
         Path path;
-        path.addRoundedRect(rect);
+        path.addRoundedRect(rect, PathRoundedRect::Strategy::PreferBezier);
```

Comment:
How can this change affect the layout of the motion-path? What exactly does the native RoundedRect behave differently from the equivalent Bézier? Also does this css also require Bézier?
```
  offset-path: inset(50px round 0%);
```
or
```
  offset-path: inset(50%);
```

Reply:
`offset-path: inset(50px round 0%);` and `offset-path: inset(50%);` are equivalent. 

But once you start using `offset-path: inset(50px round 1px);`, the starting position of the path for `PathRoundedRect::Strategy::PreferNative` goes from the top left to the bottom right. 

In the case of clip-path, it doesn't really matter because only the end shape matters, but for motion path, where the starting point of the path is matters (top left vs. bottom right)